concourse/scripts/smoke_tests_email.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eu
+
+DEPLOY_ENV=$1
+SYSTEM_DNS_ZONE_NAME=$2
+ALERT_EMAIL_ADDRESS=$3
+
+TO="${ALERT_EMAIL_ADDRESS}"
+FROM="${ALERT_EMAIL_ADDRESS}"
```

Comment:
To make it more weird, I can actually see these emails in my 'sent' tab in gmail, even though they were sent by SES....